drivers/ws281x/Makefile.dep

```diff
@@ -1,7 +1,7 @@
-FEATURES_REQUIRED_ANY += arch_avr8|arch_esp32|arch_native
+FEATURES_REQUIRED_ANY += core_avr8_mega|arch_esp32|arch_native
 
 ifeq (,$(filter ws281x_%,$(USEMODULE)))
-  ifneq (,$(filter arch_avr8,$(FEATURES_USED)))
+  ifneq (,$(filter core_avr8_mega,$(FEATURES_USED)))
```

Comment:
This is the reason for adding this feature, I assume?

Reply:
Yes, in this case the WS281X uses Mega GPIO low level drivers and code isn't same for Xmega.